localstripe/resources.py

```diff
@@ -2475,6 +2478,8 @@ def _update(self, metadata=None, items=None, trial_end=None,
                 assert proration_date > 1500000000
             if cancel_at_period_end is not None:
                 assert type(cancel_at_period_end) is bool
+            if cancel_at is not None:
+                assert type(cancel_at) is int
```

Comment:
Please add `assert cancel_at > 1500000000` (see other assertions for *date* properties).

Reply:
Done